libs/@hashintel/type-editor/src/entity-type-editor/shared/typed-values.ts

```diff
@@ -0,0 +1,31 @@
+type TupleEntry<
```

Comment:
What's up with this file?

Reply:
Needed for typedEntries in the option filtering hook